titus-ext/jooqflyway/src/main/resources/db/migration/R__0001_createTables.sql

```diff
@@ -0,0 +1,21 @@
+CREATE SCHEMA IF NOT EXISTS jobactivity;
+
+
+CREATE TABLE IF NOT EXISTS jobactivity.jobs
```

Comment:
Will gradle execute this as part of the build step?

Reply:
yes it will